dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -203,6 +203,7 @@ internal constructor(
          * to be used for each host.
          * See [DatadogInterceptor]
          */
+        @Suppress("FunctionMaxLength")
```

Comment:
I guess this is not needed? This function has the same name in the `develop` branch and everything is fine without suppression.

Reply:
for some reason detekt doesn't like this on my branch 🤔

Reply:
can you check that you running the right version with right config?

Reply:
hmmm